.travis.yml

```diff
@@ -5,5 +5,10 @@ go:
 go_import_path: github.com/aws/amazon-ecs-agent
 
 install: make get-deps
+before_script:
```

Comment:
Instead of having this just in travis, how about we add a `Makefile` target for running cyclo? Then, we can run it locally and in travis by just invoking that target. If we added something like `make static-analysis`, which invoked `go-cyclo` today and other tools in the future, all we need to do is just wire that in here.

Reply:
You are right, I should've thought of using gocyclo in development context, I focused only on adding to the PR/commit checks. Will fix as suggested.